packages/example/src/ThreePostprocessing/SwirlEffect.tsx

```diff
@@ -0,0 +1,64 @@
+import {BlendFunction, Effect} from 'postprocessing';
+import React, {forwardRef, useMemo} from 'react';
+import {Uniform, Vector2} from 'three';
+
+const fragmentShader = `
+uniform vec2 offset;
+uniform float radius;
+uniform float angle;
+
+void mainUv(inout vec2 uv) {
```

Comment:
I moved this into the example folder because most people don't need the swirl effect specifically. let's put all effects in example for now, otherwise the three package will get huge if we start shipping effects.